src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,41 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  health = 100;
+
+  constructor(name) {
+    this.name = name;
+    Animal.alive.push(this);
+  }
+
+  remove() {
+    const index = Animal.alive.indexOf(this);
+
+    if (index > -1) {
+      Animal.alive.splice(index, 1);
+    }
```

Comment:
The `remove` method uses an inefficient way to remove the animal from the `Animal.alive` array by using `indexOf` and `splice`. Instead, consider using `filter` to create a new array without the removed animal, which is generally more performant and easier to understand.